.github/workflows/ci.yml

```diff
@@ -1,29 +1,100 @@
 name: CI
 on:
   push:
-  pull_request:
   schedule:
     - cron: 13 01 * * *
 
 jobs:
-  test:
-    name: Run tests & build Docker image
+  docker-build-cache:
+    name: docker build cache
     runs-on: [linux, large, docker]
     steps:
       - uses: actions/checkout@v3.1.0
       - uses: docker/setup-buildx-action@v2.2.1
         with:
           install: true
+      - uses: docker/build-push-action@v3
+        with:
+          context: .
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+          tags: fluree/db
+          load: true
+      - name: Run tests
+        run: script/run-in-docker.sh ls
+
+  # test-nodejs:
+  #   name: test flureenjs.js
+  #   needs: docker-build-cache
+  #   runs-on: [linux, large, docker]
+  #   steps:
+  #     - uses: actions/checkout@v3.1.0
+  #     - uses: docker/setup-buildx-action@v2.2.1
+  #       with:
+  #         install: true
+  #     - uses: docker/build-push-action@v3
+  #       with:
+  #         context: .
+  #         cache-from: type=gha
+  #         cache-to: type=gha,mode=max
+  #         tags: fluree/db
+  #         load: true
+  #     - name: Run tests
+  #       run: script/run-in-docker.sh make nodejs-test
```

Comment:
Are these tests not working?

Reply:
oh whoops I should have removed this commented-out portion, `nodejs-test` got combined with another job, it's at line 81 now